src/components/utils/mapElements.jsx

```
@@ -210,240 +211,265 @@ export const MapMode = ({ mode }) => {
+          onChange={e => props.onLayerOpacityChange?.(Number(e.target.value) / 100)}
+          onMouseDown={stopPropagationOnly}
+          onMouseMove={stopPropagationOnly}
+          onTouchStart={stopPropagationOnly}
+          onTouchMove={stopPropagationOnly}
+          aria-label="Adjust layer opacity"
+        />
+      </div>
+
+      {/* Weather layer toggles */}
+      <div>
+        <label className={sectionTitleClass}>
```

ESLint

[error] A form label must be associated with a control.

(jsx-a11y/label-has-associated-control)